share/instruments/instruments.xml

```diff
@@ -5141,8 +5119,8 @@
                   <genre>marching</genre>
             </Instrument>
             <Instrument id="snare-drum">
-                  <longName>Concert Snare Drum</longName>
-                  <shortName>Con. Sn.</shortName>
+                  <longName>Snare Drum</longName>
+                  <shortName>Sn. Dr.</shortName>
```

Comment:
Same issue with mtests  as @lasconic had to fix last time

Reply:
Still an issue left with mtests, see https://travis-ci.org/musescore/MuseScore/jobs/364047450#L5030-L5059, Cornet vs. Trumpet